components/ModInputField.vue

```diff
@@ -10,18 +10,15 @@
                 :placeholder='placeholder'
                 :required='required'
                 :value="inputValue"
-                @input="event => {
-                    inputValue = event.target.value
-                    store.setInputField(label, inputValue)
-                }"
+                @input="handleInput"
```

Comment:
Nice! I was mentioning to @Anissa3005 that this should be moved to a function in the script.